torch/distributions/transforms.py

```diff
@@ -412,6 +412,29 @@ def _inverse(self, y):
         return probs.log()
 
 
+class InvertableBoltzmannTransform(Transform):
+    """
+    Transform from unconstrained space to the simplex via `y = [exp(x), 1]` and
+    then normalizing. Note that dimensionality of `y.dim() == x.dim() + 1`.
```

Comment:
I think the dimensionality change will violate an assumption in `test_distributions.py`, so you'll need special handling when adding `InvertableBoltzmannTransform` to the tests.

Reply:
nit: Can we call it `InvertibleBoltzmannTransform` instead of `InvertableBoltzmannTransform`?

Reply:
`InvertableBoltzmannTransform` -> `InvertibleBoltzmannTransform ` done.